src/platform-integration/ios/app-extensions.md

```diff
@@ -3,14 +3,27 @@ title: Adding iOS app extensions
 description: Learn how to add app extensions to your Flutter apps
 ---
 
-iOS App extensions allow you to expand functionality
+iOS app extensions allow you to expand functionality
 outside your app. Your app could appear as a home screen widget,
 or you can make portions of your app available within other apps.
 
 To learn more about app extensions, check out
 [Apple's documentation][].
 
+{{site.alert.note}}
+  If you experience a build error when building an
+  iOS app that includes an app extension, be
+  aware that there is an open bug. The workaround
+  involves changing the order of the build process.
+  For more information, check out
+  [Issue #9690][] and [Issue #135056][].
```

Comment:
**question:** Can we provide more information on the workaround? If so, can we include it on the page? It seems not user friendly to throw them to issues to figure it out.

Reply:
I disagree, Tony. The website bug concisely and clearly explains the workaround.

Reply:
Redirecting developers to the issue tracker has the added advantage that the developers will always have the latest information. Bugs have the tendency to get fixed, at which point the workaround will no longer be required.